### PR TITLE
feat: fix typing indicator

### PR DESCRIPTION
**Pull request by CalumMcCall**
#### APK
[Download build #12095](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12095/artifact/build/artifact/wire-dev-PR1876-12095.apk)
[Download build #12097](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12097/artifact/build/artifact/wire-dev-PR1876-12097.apk)
[Download build #12100](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12100/artifact/build/artifact/wire-dev-PR1876-12100.apk)
[Download build #12101](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12101/artifact/build/artifact/wire-dev-PR1876-12101.apk)
[Download build #12106](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12106/artifact/build/artifact/wire-dev-PR1876-12106.apk)
[Download build #12107](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12107/artifact/build/artifact/wire-dev-PR1876-12107.apk)
[Download build #12108](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12108/artifact/build/artifact/wire-dev-PR1876-12108.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/cursor/CursorView.scala [hunk 1]
```diff
@@ -111,12 +115,18 @@ class CursorView(val context: Context, val attrs: AttributeSet, val defStyleAttr
...
-    scrolledToBottom <- messages.scrolledToBottom
-  } yield
-    !typing && (multiline || !scrolledToBottom)
+
+  private lazy val typingUsers = for {
+    z <- zms
+    convId <- convController.currentConvId
+    userIds <- z.typing.typingUsers(convId)
+    users <- z.usersStorage.listSignal(userIds.filterNot(_ == z.selfUserId))
+  } yield users
```
Comment (JPonte) [inline app/src/main/scala/com/waz/zclient/cursor/CursorView.scala]: We're trying to move away these signal logic out of the views. Maybe put it in the CursorController?

**Comment by red-avtovo** [comment]
Still there

**Comment by CalumMcCall** [comment]
Should be fixed now

**Comment by red-avtovo** [comment]
🎉

**Comment by red-avtovo** [comment]
🎉

**Pull request merged**